### DISCUSSION TITLE
Docker container for linux/pi version

### DISCUSSION THREAD

**Discussion by fridayshoes** [GerryDazoo/Slinger#109] [Q&A]
Is it possible to run the linux/pi server in a docker container? Or has anyone already done it?

**Comment by GerryDazoo** [comment]
Yes.
 Slingbox + GerryDazoo/Slinger + Xteve + Plex Live TV (Discussion #79)

**Comment by rbf121** [comment]
I have it running in docker but it was customized for my own specific purposes (xteve and slinger in same image for use in plex).  I didn't publish a docker container but you can take a look at my Dockerfile (https://github.com/rbf121/Slinger/blob/main/Dockerfile) to build your own.  Or modify it however you want.  I have actually decided to run xteve and slinger in their own containers so I may upload a new Dockerfile to my fork at some point which just contains the slinger stuff.  I did not test it running on pi but I think xteve may need to use the arm version.

**Comment by rbf121** [comment]
I updated my fork to have a Dockerfile which only runs Slinger.  Also, published this image on docker hub if you want to just pull it down from there.
https://hub.docker.com/r/rbf121/slinger

**Comment by rbf121** [comment]
I'm not sure if anyone is using the docker image I created but if so, I just bumped the "latest" tag to match the 3.08d slinger code.

**Reply by crhiles** [reply]
I am using your docker image on my Synology. Are you able to update it to 3.08f please.

**Reply by rbf121** [reply]
This was updated a few hours back, either reset latest or take the 3.0.8.f label.
